bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewShadeHandler.java

```diff
@@ -104,8 +104,9 @@ public void initialize() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (RefreshType.REFRESH.equals(command)) {
+        if (RefreshType.REFRESH == command) {
             requestRefreshShadePosition();
+            requestRefreshShadeBatteryLevel();
```

Comment:
The REFRESH command should be at channel level, this is not a global command for all channels. 
Are all of the channels depending on each other?

Reply:
You are absolutely right - nice catch! I have updated the logic so that channel is taken into consideration. Documentation has been updated accordingly to clarify how it works.